### DISCUSSION TITLE
How to autohide third party plugin view toolbar buttons

### DISCUSSION THREAD

**Discussion by kookma** [NicolasPetton/Notebook#5] [Q&A] [answered]
- Notebook very nicely autohides all view toolbar buttons except **close** tiddler
- Installing third party plugins like my favorites plugin (https://github.com/kookma/TW-Favorites) adds extra view toolbar buttons
- Is it possible Notebook automatically autohide those buttons? Or simply in customization tiddler explain how to do this (part of customization)?

**Screenshot**
![image](https://user-images.githubusercontent.com/830394/102449874-71723600-404a-11eb-99fa-f93ff8bca951.png)

**Off topic**: Very nice status badge, I love it!

**Comment by NicolasPetton** [comment]
It should work automatically, I'll see if I can fix it.

**Comment by NicolasPetton** [comment, marked as answer]
@kookma I made an issue out of this https://github.com/NicolasPetton/Notebook/issues/7

**Reply by NicolasPetton** [reply]
@kookma the issue was fixed in https://github.com/NicolasPetton/Notebook/commit/eb4dc3e33497623358608cfb44b0d7b29ab333ee

**Reply by kookma** [reply]
@NicolasPetton - Many thanks, this works now

**Reply by NicolasPetton** [reply]
@kookma I think there's a way in GitHub Discussions to mark a discussion as answered. Could you do that?

**Reply by kookma** [reply]
@NicolasPetton - I had to change the category to Q&A then I was able to select the best answer!

**Reply by NicolasPetton** [reply]
Thanks for letting me know.

**Reply by NicolasPetton** [reply]
Would you mind doing the same thing for other resolved discussions?

**Reply by kookma** [reply]
Sure, I just change their category! As the answered button only is seen on Q&A category

**Reply by kookma** [reply]
There was one other I moved to Q&A and Answered. Two others are ideas!
